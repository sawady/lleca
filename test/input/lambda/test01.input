(\ x y. y (x x y)) (\ x y. y (x x y))
