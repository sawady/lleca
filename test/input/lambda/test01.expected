App(
  Lam(
    x,
    Lam(
      y,
      App(
        Var(
          y
        ),
        App(
          App(
            Var(
              x
            ),
            Var(
              x
            )
          ),
          Var(
            y
          )
        )
      )
    )
  ),
  Lam(
    x,
    Lam(
      y,
      App(
        Var(
          y
        ),
        App(
          App(
            Var(
              x
            ),
            Var(
              x
            )
          ),
          Var(
            y
          )
        )
      )
    )
  )
)
